lib/rails_response_dumper/runner.rb

```diff
@@ -67,11 +70,13 @@ def run_dumps
           begin
             rollback_after do
               t0 = Time.now
+              defined.config&.before_all_block&.call
```

Comment:
Although this is named "before all" it is actually doing the same things as before "each" (as in, each dump method).

For this to be before "all", I'd expect it ot be outside the `dumper_blocks.each` loop.